dependency-check-suppressions.xml

```diff
@@ -1,130 +1,29 @@
         <cve>CVE-2020-9488</cve>
-    </suppress>
-    <suppress>
-        <notes>
-            <![CDATA[CVE is a browser vulnerability]]>
-        </notes>
+        <cve>CVE-2020-11612</cve>
+        <cve>CVE-2017-18640</cve>
         <cve>CVE-2020-5421</cve>
-    </suppress>
-    <suppress until="2020-11-30Z">
-        <notes>
-            <![CDATA[file name: tomcat-embed-core-9.0.39.jar]]>
-        </notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
-        <cve>CVE-2020-13943</cve>
-    </suppress>
-    <suppress until="2020-11-30Z">
-        <notes>
-            <![CDATA[file name: tomcat-embed-websocket-9.0.39.jar]]>
-        </notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
+        <cve>CVE-2018-1258</cve>
+        <cve>CVE-2020-5407</cve>
+        <cve>CVE-2020-5408</cve>
+        <cve>CVE-2019-17569</cve>
+        <cve>CVE-2020-11996</cve>
+        <cve>CVE-2020-13934</cve>
+        <cve>CVE-2020-13935</cve>
         <cve>CVE-2020-13943</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: httpclient-4.5.10.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.httpcomponents/httpclient@.*$</packageUrl>
-        <cpe>cpe:/a:apache:httpclient</cpe>
-        <cve>CVE-2020-13956</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: junit-4.12.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/junit/junit@.*$</packageUrl>
-        <vulnerabilityName>CVE-2020-15250</vulnerabilityName>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: jackson-databind-2.10.0.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
-        <cpe>cpe:/a:fasterxml:jackson</cpe>
-        <cpe>cpe:/a:fasterxml:jackson-databind</cpe>
-        <cve>CVE-2020-25649</cve>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-core-9.0.39.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-core@.*$</packageUrl>
         <cve>CVE-2020-17527</cve>
-        <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
-        <cpe>cpe:/a:apache_software_foundation:tomcat</cpe>
-        <cpe>cpe:/a:apache:tomcat</cpe>
-    </suppress>
-    <suppress>
-        <notes><![CDATA[
-   file name: tomcat-embed-websocket-9.0.39.jar
-   ]]></notes>
-        <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
-        <cve>CVE-2020-17527</cve>
-        <cpe>cpe:/a:java-websocket_project:java-websocket</cpe>
-        <cpe>cpe:/a:apache_tomcat:apache_tomcat</cpe>
-        <cpe>cpe:/a:apache_software_foundation:tomcat</cpe>
-        <cpe>cpe:/a:apache:tomcat</cpe>
-    </suppress>
-    <suppress>
-        <cve>CVE-2020-8908</cve>
```

Comment:
wouldnt removing all the checks break something (code wise)?

Reply:
Many no longer apply. I've removed them all and re-added the ones that are relevant